src/shared/lib/alterationCountHelpers.ts

```diff
@@ -74,12 +88,22 @@ export function getAlterationCountsForCancerTypesForAllGenes(
     } = _.mergeWith({}, ...flattened, customizer) as {
         [uniqueSampleKey: string]: ExtendedAlteration[];
     };
-    return countAlterationOccurences(
-        samplesByCancerType,
-        merged,
-        molecularProfileIdsByAlterationType,
-        coverageInformation
-    );
+
+    const alterationCounts = countByProperty
```

Comment:
`countByProperty` is not a boolean so we should do something like this, right?

```suggestion
    const alterationCounts = countByProperty === 'sampleCounts'
```